tests/dags/test_model_dags.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm not a big fan of having test DAGs in separate files, especially since the file name does not match the DAG name, making it harder to find out where a DAG lives. Can you rename your file to the dag name and  refer to it inside the tests where you are using it?